src/pages/ni/config/RhConfig.vue

```diff
@@ -493,6 +553,61 @@ export default {
       if (!obj.name.required) return REQUIRED_LABEL;
       else if (!obj.name.sector) return 'Nom déjà existant';
     },
+    async getAdministrativeDocuments () {
+      try {
+        this.administrativeDocuments = await AdministrativeDocument.list();
+      } catch (e) {
+        console.error(e);
```

Comment:
il faut ajouter `this.administrativeDocuments = []` pour qu'il soit toujours bien defini